# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/midi/MIDIPort.h
# Commit: 225adb498b09
# Full Hash: 225adb498b091a93ddf7081c1f4356f705594a7b
# Author: Gabriele Svelto <gsvelto@mozilla.com>
# Date: 2022-04-12 15:57:11
# Regressor Bug: 1758468
# File Overlap Count: 1
# Description:
#   Bug 1758468 - Use stable, anonymous IDs for Web MIDI ports r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142550
# ==============================================================================

diff -r 3c6e6d0d7b38 -r 225adb498b09 dom/midi/MIDIPort.h
--- a/dom/midi/MIDIPort.h	Tue Apr 12 12:31:27 2022 +0000
+++ b/dom/midi/MIDIPort.h	Tue Apr 12 12:31:27 2022 +0000
@@ -73,6 +73,7 @@
   IMPL_EVENT_HANDLER(statechange)
 
   void DisconnectFromOwner() override;
+  const nsString& StableId();
 
  protected:
   // IPC Actor corresponding to this class